### PR TITLE
Working frontend with new backend

### PR DESCRIPTION
**Pull request by therishidesai**
- Sends correct JSON data to the server
- Redirects user to the jupyter url when it receives it from the server

### CODE REVIEW

**Review by narendasan — COMMENTED** [review]
Redirect works but I dont get a token, what is the intended behavior here?

**Review by narendasan — COMMENTED** [review]
Have some sort of visual sign that a container is spinning up. (e.g. a loading page)

**Comment by therishidesai** [comment]
Was the flask server running when you requested a token?

**Comment by therishidesai** [comment]
So the whole workflow should work properly now. To test it make sure the backend server is running also and I have also disabled the images that are not working.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM node:boron
```
Comment (tyler-thetyrant) [inline Dockerfile]: get rid of this dockerfile

**Comment by therishidesai** [comment]
This is should be good to merge now.

**Review by narendasan — APPROVED** [review]

**Pull request merged**